addons/web/static/tests/helpers/mock_server.js

```diff
@@ -453,6 +461,27 @@ export class MockServer {
         };
     }
 
+    _editableNode(node, modelName) {
+        if (node.tagName === "form") {
```

Comment:
not really important, but I personally think `switch` statements are nicer for that kind of code:
```js
switch (node.tagName) {
  case "form":
    return true;
  case "tree:
    ...
  case "field:
    ..
  default:
    return false;
}
```